src/styles/colors.scss

```diff
@@ -39,3 +39,10 @@
 // Default //////////////////////////
 
 body { color: var(--color-text); }
+
+
+// Other Colors ////////////////
+:root {
+  // Temporarily added
```

Comment:
Why is it temporarily added? When will we be able to remove it?

Reply:
Oh okay! It's because we haven't decided yet how we gonna add this specific tokens. I'll will wait for Renatinha 👍